src/core/core.tooltip.js

```diff
@@ -810,9 +811,13 @@ var exports = Element.extend({
 			ctx.fillStyle = textColor;
 			helpers.each(bodyItem.before, fillLineOfText);
 
-			helpers.each(bodyItem.lines, function(line) {
+			len = bodyItem.lines.length;
+			for (j = 0; j < len; j++) {
+				var line = bodyItem.lines[j];
+				var firstLine = j === 0;
+
 				// Draw Legend-like boxes if needed
-				if (drawColorBoxes) {
+				if (drawColorBoxes && firstLine) {
```

Comment:
I would make this `if (drawColorBoxes && j === 0) {` and remove `firstLine` variable

Reply:
done